spec/04-basic-concepts.md

```diff
@@ -1309,7 +1309,8 @@ three kinds: automatic, static, and allocated.
 
 A variable having *automatic storage duration* comes into being and is
 initialized at its declaration or on its first use, if it has no
-declaration. Its lifetime is delimited by an enclosing scope ([§§](#scope)). The
+declaration (notice that this is simplified and not always true, please refer to [§§](07-variables.md#local-variables) for more information).
```

Comment:
This is kind of vague. If we mention it's not true, then we have to say when. Otherwise, it should be just "for more details, see ...".